### PR TITLE
Adds super k to json quickstart behind feature flag

### PR DESCRIPTION
**Pull request by dakota002**

### CODE REVIEW

**Review by Vidushi-GitHub — COMMENTED** [review]
We don't have PR for SK schema yet.

Two different names look inconsistent, although one is notices topic other is for mission. 
Is topic name already chosen?

**Comment by dakota002** [comment]
There are two, I just replied to them on Zendesk. I will update this PR shortly

File: app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx [hunk 1]
```diff
@@ -232,7 +232,10 @@ export function NoticeTypeCheckboxes({
   }
 
   if (useFeature('SUPER_K_QUICKSTART')) {
-    JsonNoticeTypes['Super Kamiokande'] = ['gcn.notices.superk']
+    JsonNoticeTypes['Super Kamiokande'] = [
+      'gcn.notices.superk.sn_alert.real',
```
[the remark this answers is not in the document]
Comment (dakota002) [reply app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx:236]: Do we want them to use the real/test option as a property in the alert?

[the remark this answers is not in the document]
Comment (Vidushi-GitHub) [reply app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx:236]: Other missions are using only alert.
I remember we have asked a few missions to create .test topic for producers check.

[the remark this answers is not in the document]
Comment (lpsinger) [reply app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx:236]: I like @Vidushi-GitHub's suggestion. Are we going to accept that patch?

[the remark this answers is not in the document]
Comment (dakota002) [reply app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx:236]: It's all fine by me, as long as the Super K team is aware

**Pull request merged**